Makefile

```diff
@@ -1,4 +1,7 @@
-########################################
+- run:
+	    name: Build portable binary
+	    command: |
+	        make macos-portable
```

Comment:
is this an accident

Reply:
For some reason I think it was removed upstream? Lemme check again though

Reply:
Oh nvm yeah

Reply:
Yes just removed it